VideoLocker/src/main/java/org/edx/mobile/util/images/ErrorUtils.java

```diff
@@ -16,31 +17,43 @@
 
     protected static final Logger logger = new Logger(ErrorUtils.class.getName());
 
+    public static class Error {
+        public MessageType type;
+        public String message;
+    }
```

Comment:
There is a case where we just need the error message based on an exception here:
`org.edx.mobile.profiles.UserProfilePresenter.ViewInterface#showError`